packages/template-retail-react-app/app/components/store-locator-modal/stores-list.jsx

```diff
@@ -15,78 +15,113 @@ import {
-                                    defaultMessage: 'away'
-                                })}
-                            </Box>
-                        </>
-                    ) : (
-                        ''
-                    )}
-                    {store.phone !== undefined ? (
-                        <>
-                            <br />
-                            <Box fontSize="md" color="gray.600">
-                                {intl.formatMessage({
-                                    id: 'store_locator.description.phone',
-                                    defaultMessage: 'Phone:'
-                                })}{' '}
-                                {store.phone}
-                            </Box>
-                        </>
-                    ) : (
-                        ''
-                    )}
-                    {store?.storeHours ? (
-                        <>
-                            {' '}
-                            <AccordionButton color="blue.700" style={{marginTop: '10px'}}>
-                                <Box fontSize="lg">
-                                    {intl.formatMessage({
-                                        id: 'store_locator.action.viewMore',
-                                        defaultMessage: 'View More'
-                                    })}
-                                </Box>
-                                <AccordionIcon />
-                            </AccordionButton>
-                            <AccordionPanel mb={6} mt={4}>
-                                <div
-                                    dangerouslySetInnerHTML={{
-                                        __html: store?.storeHours
-                                    }}
-                                />
-                            </AccordionPanel>{' '}
-                        </>
-                    ) : (
-                        ''
-                    )}
-                </Box>
-            </AccordionItem>
+    useEffect(() => {
+        setSelectedStore(JSON.parse(window.localStorage.getItem(storeInfoKey))?.id || '')
+    }, [storeInfoKey])
+
+    const handleChange = (storeId) => {
+        setSelectedStore(storeId)
+        const store = storesInfo.find((store) => store.id === storeId)
+        window.localStorage.setItem(
+            storeInfoKey,
+            JSON.stringify({
+                id: storeId,
+                name: store.name || null,
+                inventoryId: store.inventoryId || null
+            })
         )
-    })
+    }
+
+    return (
+        <RadioGroup onChange={handleChange} value={selectedStore}>
+            {storesInfo?.map((store, index) => {
+                return (
+                    <AccordionItem key={index}>
+                        <HStack align="flex-start" mt="16px" mb="16px">
+                            <Radio
+                                value={store.id}
+                                mt="1.5px"
```

Comment:
(non-blocking) weird to have `1.5px` which seems like not a valid value. Reminds me of when Retina displays first came out and there were weird hacks like ~10 years ago. But if it's correct, then fine

Reply:
ooh good callout! So dug into this a bit, seems like all modern browsers support this while older browsers may round to the nearest whole number.

I also don't see anywhere else in PWA Kit that uses this so just for consistency i'll change it to `1px`. UI wise this puts the radio button bottom-aligned with the store name rather than centered which isn't a big deal